crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -122,19 +122,21 @@ pub struct AuthServerConfig {
     pub(crate) socket_addr: SocketAddr,
     /// The secrete for the auth layer of the server.
     pub(crate) secret: JwtSecret,
+    /// The maximum response size for the server.
+    pub(crate) rpc_max_response_size: u32,
```

Comment:
I had another look at:

https://github.com/paradigmxyz/reth/blob/4d3d4974a48b4e838ccda50c3171da11c4885a13/crates/rpc/rpc-builder/src/lib.rs#L944-L946

and I'd like to replicate this for `AuthServerConfig`.

so basically:
1. a field `server_{config|builder}` 
2. then a separate function just like:

https://github.com/paradigmxyz/reth/blob/4d3d4974a48b4e838ccda50c3171da11c4885a13/crates/rpc/rpc-builder/src/lib.rs#L995-L1003